envision/utils/multiprocessing_queue.py

```diff
@@ -25,6 +25,7 @@
 # Please refer to https://eli.thegreenplace.net/pages/about for more info about the author
 
 import multiprocessing
+import multiprocessing.queues
```

Comment:
So we need both `import multiprocessing` and `import multiprocessing.queues`?

Reply:
yes, we use both for this implementation.

Reply:
`multiprocessing.queues` is used to access `multiprocessing.queues.Queue` specifically.